src/main/steps/common/common.content.ts

```diff
@@ -1,32 +1,37 @@
 const en = {
   phase: 'Beta',
-  feedback: 'This is a new service – your <a class="govuk-link" aria-label="Feedback link, This will open a new tab. You’ll need to return to this tab and continue with your application within 60 mins so you don’t lose your progress." href="https://www.smartsurvey.co.uk/s/Divorce_Feedback" target="_blank">feedback</a> will help us to improve it.',
+  feedback:
+    'This is a new service – your <a class="govuk-link" aria-label="Feedback link, This will open a new tab. You’ll need to return to this tab and continue with your application within 60 mins so you don’t lose your progress." href="https://www.smartsurvey.co.uk/s/Divorce_Feedback" target="_blank">feedback</a> will help us to improve it.',
```

Comment:
dont really see the point of this being on a new line as it will still be quite long.

Reply:
Prettier by default is configured to wrap anything greater than a line length of `80`, this can be [configured](https://prettier.io/docs/en/options.html#print-width), but then would go against the wrapping you mentioned above.